redaxo/src/core/lib/console/db/set_connection.php

```diff
@@ -15,11 +15,12 @@ class rex_command_db_set_connection extends rex_console_command
     protected function configure()
     {
         $this
-            ->setDescription('Sets database connection credentials')
+            ->setDescription("Sets database connection credentials.\n  Checks by default if a database connection can be established with the new settings.")
```

Comment:
Warum die spaces vor „ Checks“?

Reply:
schaut ohne etwas unformatiert aus.
ohne:
![image](https://user-images.githubusercontent.com/21292755/71772527-ac1fb300-2f4c-11ea-9e4c-847afc32bddf.png)
mit:
![image](https://user-images.githubusercontent.com/21292755/71772523-99a57980-2f4c-11ea-917a-b5d92b503c1b.png)

Reply:
Macht das symfony ggf. das ganze von haus aus schön wenn man kein explizites newline notiert?

Reply:
ist eher so meh
![image](https://user-images.githubusercontent.com/21292755/71772561-5b5c8a00-2f4d-11ea-9272-3507768e2f94.png)

kann mich aber mit beiden (mit spaces oder ohne umbruch) anfreunden.

Reply:
Ggf hat gregor ne idee dazu

Reply:
Ich glaube da sind Newlines nicht wirklich vorgesehen, sollte nur eine kurze Beschreibung sein.
Die wird auch in der Liste der Commands verwenden (bei Aufruf von `redaxo/bin/console` ohne Command).
Das sieht deine aktuelle Variante vermutlich komisch aus.
Würde es daher wohl einzeilig machen.

Reply:
Eventuell lässt man hier auch den zweiten Satz weg, und nutzt `setHelp`.
Hier ein Beispiel von Symfony:
https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Command/SecretsSetCommand.php

Reply:
habs rausgenommen und mit `setHelp` angehangen. Sah wie du sagtest in der Auflistung der Commands komisch aus.